velox/expression/fuzzer/ExpressionFuzzerTest.cpp

```diff
@@ -33,18 +34,37 @@ DEFINE_int64(
     "Initial seed for random number generator used to reproduce previous "
     "results (0 means start with random seed).");
 
+DEFINE_string(
```

Comment:
can you also update the `fuzzer.rst' with details about these changes and the startup flags